### PR TITLE
[MISPV3] mypy ignore

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
Related: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-10508)

## Description
ignore mypy errors in `MISPV3.py`

### CODE REVIEW

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 1]
```diff
@@ -561,7 +561,7 @@ def create_event_command(demisto_args: dict):
 
     if isinstance(new_event, dict) and new_event.get('errors'):
         raise DemistoException(new_event.get('errors'))
-
+    assert isinstance(new_event, MISPEvent), "The new event is not an instance of MISPEvent"
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:564]: @dorschw  do we allow `assert` in content?

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 2]
```diff
@@ -679,8 +679,10 @@ def add_attribute(event_id: int = None, internal: bool = False, demisto_args: di
         if not response:
             raise DemistoException(
                 f"Error: An event with the given id: {event_id} was not found in MISP. please check it once again")
-        new_event = response[0]  # response[0] is MISP event
+        new_event = response[0]  # type: ignore[assignment]
+        # response[0] is MISP event
 
+    assert isinstance(new_event, MISPEvent), "The new event is not an instance of MISPEvent"
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:685]: same issue

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 3]
```diff
@@ -1311,8 +1313,8 @@ def remove_tag(demisto_args: dict, is_attribute=False):
     Function will remove tag to given UUID of event or attribute.
     is_attribute (bool): if the given UUID is an attribute's one. Otherwise it's event's.
     """
-    uuid = demisto_args.get('uuid')
-    tag = demisto_args.get('tag')
+    uuid = demisto_args['uuid']
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:1316]: are we sure about changing this?

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 4]
```diff
@@ -1662,7 +1664,7 @@ def delete_attribute_command(demisto_args: dict) -> CommandResults:
     """
     Gets an attribute id and deletes it.
     """
-    attribute_id = demisto_args.get('attribute_id')
+    attribute_id = demisto_args['attribute_id']
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:1667]: same issue

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Packs/MISP/Integrations/MISPV3/MISPV3.py (hunk 1 above)
Comment (dorschw) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py:564]: in rare cases, @shmuel44 let's discuss this one offline

File: Packs/MISP/Integrations/MISPV3/MISPV3.py (hunk 3 above)
Comment (dorschw) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py:1316]: in this particular case, `uuid` is `required`

File: Packs/MISP/Integrations/MISPV3/MISPV3.py (hunk 4 above)
Comment (dorschw) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py:1667]: @shmuel44 if you verified each and every arg we changed `.get` to`[]` is required, I'm ok with that.

**Review by dorschw — APPROVED** [review]

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 5]
```diff
@@ -559,9 +559,10 @@ def create_event_command(demisto_args: dict):
     new_event = get_new_misp_event_object(demisto_args)
     new_event = PYMISP.add_event(new_event, True)
 
-    if isinstance(new_event, dict) and new_event.get('errors'):
-        raise DemistoException(new_event.get('errors'))
-
+    if isinstance(new_event, dict):
+        if new_event.get('errors'):
+            raise DemistoException(new_event.get('errors'))
+        raise DemistoException("Unknown error occurred while creating event.")
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:565]:
```suggestion
        raise DemistoException(f"Unknown event type:{type(new_event)}.")
```

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 6]
```diff
@@ -559,9 +559,10 @@ def create_event_command(demisto_args: dict):
     new_event = get_new_misp_event_object(demisto_args)
     new_event = PYMISP.add_event(new_event, True)
 
-    if isinstance(new_event, dict) and new_event.get('errors'):
-        raise DemistoException(new_event.get('errors'))
-
+    if isinstance(new_event, dict):
+        if new_event.get('errors'):
+            raise DemistoException(new_event.get('errors'))
```
Comment (kobymeir) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:564]:
```suggestion
            raise DemistoException(f"Errors:{new_event.get('errors')}")
```

**Review by kobymeir — CHANGES_REQUESTED** [review]

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 7]
```diff
@@ -583,7 +584,7 @@ def add_user_to_misp(demisto_args: dict = {}):
         demisto_args (dict): Demisto args
     """
     new_user = MISPUser()
-    new_user.email = demisto_args.get('email')
+    new_user.email = demisto_args['email']
```
[the remark this answers is not in the document]
Comment (shmuel44) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py:587]: yes

**Review by kobymeir — APPROVED** [review]

**Comment by shmuel44** [comment]
@kobymeir  Needs to be force merged as it failed on DO106, and we are unable to update this due to [CIAC-11645](https://jira-dc.paloaltonetworks.com/browse/CIAC-11645).

**Pull request merged**